dev/containers/docker-compose.yml

```diff
@@ -43,21 +40,22 @@ services:
   #     - wiki
 
   wiki:
+    container_name: wiki-app
     build:
       context: ../..
       dockerfile: dev/containers/Dockerfile
     depends_on:
       - db
-    networks:
-      - wikinet
     ports:
       - "3000:3000"
     volumes:
-      - ../..:/wiki
-      - /wiki/node_modules
+      - ../../client:/wiki/client
+      - ../../server:/wiki/server
+      - ../../dev:/wiki/dev
+      - ../../package.json:/wiki/package.json
+      - ../../yarn.lock:/wiki/yarn.lock
+      - ../../.babelrc:/wiki/.babelrc
```

Comment:
You're missing a few files here. All files at the root of the project should also be mounted.

Reply:
Ah true, I thought the others were unnecessary but I see now that they won't appear in the VS Code project unless listed here. Bummer, was hoping to avoid that. I'll try
```
- ../..:/wiki
- /wiki/node_modules
- /wiki/.git

Reply:
Cool, the above works. Cleaner than my initial solution anyway!